docs/code-snippets/excel-snippets.yaml

```
@@ -381,3 +409,18 @@
       // Print the value of A1.
       console.log(range.getValue());
     }
+'ExcelScript.Worksheet#setTabColor:member(1)':
+  - |-
+    /**
+     * This script sets the tab color of every worksheet in the workbook to red.
+     */
+    function main(workbook: ExcelScript.Workbook) {
+      // Get all the worksheets in the workbook. 
+      let sheets = workbook.getWorksheets();
+      
+      // Set the tab color each worksheet to a random color
```

Or `Set the tab color of each worksheet to red.` then you can delete the next comment (line 423).
```suggestion
      // Set the tab color of each worksheet to a random color.
```